tests/sentry/incidents/endpoints/test_project_alert_rule_index.py

```diff
@@ -541,3 +541,134 @@ def test_offset_pagination(self):
 
         self.assert_alert_rule_serialized(self.two_alert_rule, result[0], skip_dates=True)
         self.assert_alert_rule_serialized(self.yet_another_alert_rule, result[1], skip_dates=True)
+
+
+@freeze_time()
+class AlertRuleCreateEndpointTestCrashRateAlert(APITestCase):
+    endpoint = "sentry-api-0-project-alert-rules"
+    method = "post"
+
+    def setUp(self):
+        super().setUp()
+        self.valid_alert_rule = {
+            "aggregate": "crash_free_percentage()",
+            "query": "",
+            "timeWindow": "60",
+            "resolveThreshold": 90,
+            "thresholdType": 1,
+            "triggers": [
+                {
+                    "label": "critical",
+                    "alertThreshold": 70,
+                    "actions": [
+                        {"type": "email", "targetType": "team", "targetIdentifier": self.team.id}
+                    ],
+                },
+                {
+                    "label": "warning",
+                    "alertThreshold": 80,
+                    "actions": [
+                        {"type": "email", "targetType": "team", "targetIdentifier": self.team.id},
+                        {"type": "email", "targetType": "user", "targetIdentifier": self.user.id},
+                    ],
+                },
+            ],
+            "projects": [self.project.slug],
+            "owner": self.user.id,
+            "name": "JustAValidTestRule",
+            "dataset": "sessions",
+            "eventTypes": ["session"],
+        }
+
+    @fixture
+    def organization(self):
+        return self.create_organization()
+
+    @fixture
+    def project(self):
+        return self.create_project(organization=self.organization)
+
+    @fixture
+    def user(self):
+        return self.create_user()
+
+    def test_simple_crash_rate_alerts_for_sessions(self):
+        # Login
+        self.create_member(
+            user=self.user, organization=self.organization, role="owner", teams=[self.team]
+        )
+        self.login_as(self.user)
+
+        with self.feature(["organizations:incidents", "organizations:performance-view"]):
```

Comment:
When I omit that feature the test fail, shouldn't this https://github.com/getsentry/sentry/blob/master/src/sentry/features/__init__.py#L122 have a default `True` for it to be enabled by default?

Reply:
Made the mistake of looking at https://github.com/getsentry/sentry/blob/520197a848a368c7beb30b16047e3b2076dff740/src/sentry/conf/server.py#L913 instead of the features manager